content/07.pathogenesis.md

```diff
@@ -124,42 +124,42 @@ The mechanism behind these insults has been suggested to be related to inflammat
-<!-- Suggestion: define distinction between Cytokine Release Syndrome and Cytokine Storm Syndrome-->
-One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, which may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
-In general, the interferons (IFNs) activate intracellular antimicrobial programs and are the major first line of defense against viruses [@doi:10.1016/j.chom.2020.05.008].
-IFNs are a type of cytokines along with chemokines, interleukins (IL), lymphokines and tumor necrosis factor (TNF).
-Cytokines can be classified based on the nature of the immune response or based on their specific roles, as reviewed by [@doi:10.1016/j.bbamcr.2014.05.014]).
-I@doi:10.1016/j.immuni.2020.06.017] describe the concept of the cytokine storm in detail, although [@doi:10.1001/jamainternmed.2020.3313] claim that the widespread use of the term can be misleading.
--->
+#### Cytokine Release Syndrome
+
+Symptoms of a disease can be caused by a pathogen, but they can also be caused by the immune system's reaction to the pathogen.
+A dysregulated immune response can cause significant damage to the host [@isbn:9780815332183; @isbn:9780071283663; @doi:10.1016/j.chom.2020.05.009].
+The inflammatory response has received particular attention for its role in both a healthy response to infection and a pathogenic one.
+Inflammation is one of the most visible components of the immune response, as it is responsible for hallmarks of injury such as pain, swelling, etc. [@doi:10.18632/oncotarget.23208].
+In response to injury or to signaling by PRRs upon detection of a pathogen, the immune system stimulates leukocytes that travel to the site of the threat, where they then produce cytokines [@doi:10.18632/oncotarget.23208].
+Cytokines are a diverse group of small proteins that play an important role in intercellular signaling [@doi:10.1128/MMBR.05015-11].
+This category includes interleukins, which are produced by and regulate the development of leukocytes, interferons, which play an antiviral and antiproliferative role in responding to viral threat, and tumor necrosis factors, which activate cytotoxic T-lymphocytes [@doi:10.1128/MMBR.05015-11; @doi:10.1097/AIA.0b013e318034194e].
+Cytokines can be both pro- and anti-inflammatory, which means they can either stimulate or inhibit the production of additional cytokines [@doi:10.1097/AIA.0b013e318034194e].
+Dysregulation is typically discussed in terms of excessive pro-inflammatory cytokine activity unchecked by the anti-inflammatory counterparts, which can therefore cause a cascade of the inflammatory response [@doi:10.1159/000087104].
```

Comment:
Would it make sense to add a sentence about how persistent levels of inflammation can damage cells, which is why its dangerous